### DISCUSSION TITLE
Estimated number of cells vs ArcR detection

### DISCUSSION THREAD

**Discussion by e-arslan** [GreenleafLab/ArchR#1125] [Questions / Documentation] [answered]
I've been using ArcR for many scATAC data analyses and for the first time I see a huge difference between web_summary estimated number of cells and the ArcR first detected cells. 

Here is an example, I get the following estimated number of cells at 10x cellranger output web_summary:
X: 105
Y: 1819
Z: 108
T: 1162

and after creating the arrow files via fragments data for each before any doublet detection, here are the number of cells:
X: 170
Y: 5893
Z: 138
T: 559

I'm curious why I'm getting that huge difference. Especially for the Y condition.

ps: Here is after doublet detection:
X: 170
Y: 5546
Z: 130
T: 556

ArchR version is ‘1.0.2’

**Comment by rcorces** [comment]
Please note that this is not a bug or an error but rather a usage question and should have been posted in the Discussions Forum.

I have moved it there.

**Comment by rcorces** [comment, marked as answer]
I dont think there is much correlation between what CellRanger thinks is a pass filter cell and what actually is a pass filter cell. I wouldnt trust the cell ranger estimates at all.

**Reply by e-arslan** [reply]
I see, thanks a lot.

**Comment by nvribeiro** [comment]
Hi, I have the same question... I understand that cellranger might have different pass filters and therefore provide a different estimation from ArchR. But in my case, the experiment was done loading ~10k cells in the 10X chip and cellranger tells me there are an estimated number of cells = 9,756 but after createArrowFiles ArchR says 18,322 cells pass filter. That is a huge difference and looks weird to me, any thoughts on that?
